src/main.js

```diff
@@ -156,20 +163,33 @@ function toggleMakePoster() {
 function toggleShowSavedPoster() {
   viewMainPoster.classList.toggle("hidden");
   viewShowSavedPosters.classList.toggle("hidden");
+  addMiniPosters();
 }
 
 function clickShowMyPoster() {
+  event.preventDefault();
```

Comment:
Should this function have `event` as the first parameter? Or is this event global?

Reply:
Yes. We added `event` parameter to keep this event local.